resources/views/regulated-organizations/edit.blade.php

```diff
@@ -135,7 +137,10 @@
 
                 <div class="field">
```

Comment:
Does this need the error class? e.g. `@error('website_link') field--error @enderror"`